Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/Helpers/SmartWeakEvent.cs

```diff
@@ -0,0 +1,174 @@
+﻿// ******************************************************************
```

Comment:
You're right...But then the WeakEvent should be moved to Helpers as it could be useful even outside the markdowntextblock:)